app/src/main/res/values/strings.xml

```diff
@@ -1,4 +1,18 @@
 <resources>
     <string name="app_name">UnoHouseAndroid</string>
     <string name="action_settings">Settings</string>
+    <string name="title_activity_login">Sign in</string>
+
+    <!-- Strings related to login -->
+    <string name="prompt_login">Login</string>
+    <string name="prompt_password">Password</string>
+    <string name="action_sign_in">Sign in or register</string>
+    <string name="action_sign_in_short">Sign in</string>
+    <string name="error_invalid_login">This login is invalid</string>
+    <string name="error_invalid_password">This password is too short</string>
+    <string name="error_incorrect_password">This password is incorrect</string>
+    <string name="error_field_required">This field is required</string>
+    <string name="permission_rationale">"Contacts permissions are needed for providing email
```

Comment:
Po tym wierszu wnioskuję że możliwe będzie auto uzupełnianie maila z książki telefonicznej, Tak?
Czy zapewni nam to mozliwosć zapamiętywania sesji? czyli bycie cały czas zalogowanym, korzystajac z naszego konta?
Czy w takim wypadku nie powinniśmy rozważyć też rejestracji?